cmd/tx/tx_utils/tx_utils.go

```diff
@@ -15,10 +15,10 @@ func CheckTxStdOut(stdout bytes.Buffer) error {
 		return err
 	}
 
-	if strings.Contains(response.RawLog, "fail") || strings.Contains(response.RawLog, "error") {
+	if strings.Contains(response.RawLog, "fail") || strings.Contains(response.RawLog, "error") ||
+		strings.Contains(response.RawLog, "insufficient funds") {
```

Comment:
IMO it's better to have  a more generic way to catch all non-zero code responses and simply return the raw message vs handling each message separately

Reply:
Gotcha. I thought you meant to the terminal exit code.